shell/platform/darwin/common/framework/Headers/FlutterChannels.h

```diff
@@ -143,9 +143,46 @@ FLUTTER_DARWIN_EXPORT
  * Adjusts the number of messages that will get buffered when sending messages to
  * channels that aren't fully set up yet.  For example, the engine isn't running
  * yet or the channel's message handler isn't set up on the Dart side yet.
+ *
+ * @param name The channel name.
+ * @param messenger The binary messenger.
+ * @param newSize The new size of the buffer.
+ */
++ (void)resizeChannelWithName:(NSString*)name
+              binaryMessenger:(NSObject<FlutterBinaryMessenger>*)messenger
+                      newSize:(NSInteger)newSize;
+
+/**
+ * Adjusts the number of messages that will get buffered when sending messages to
+ * channels that aren't fully set up yet.  For example, the engine isn't running
+ * yet or the channel's message handler isn't set up on the Dart side yet.
+ *
+ * @param newSize The new size of the buffer.
  */
 - (void)resizeChannelBuffer:(NSInteger)newSize;
 
+/**
+ * Toggles whether the channel should show warning messages when discarding messages
```

Comment:
If all this controls is warnings, the naming of the method seems very confusing. I would expect allowing overflow to mean it actually changes the behavior of handling extra messages.

Reply:
I stuck to the naming from channels_buffer.dart. 
https://github.com/flutter/engine/blob/93e8901490e78c7ba7e319cce4470d9c6478c6dc/lib/ui/channel_buffers.dart#L574

I agree that it is confusing. Is it ok to use a different naming? If yes, I would switch to `expectOverflow` or `warnOnOverflow`.